test/util/bitcoin-util-test.py

```diff
@@ -63,87 +67,93 @@ def bctester(testDir, input_basename, buildenv):
     else:
         sys.exit(0)
 
-def bctest(testDir, testObj, buildenv):
+
+def bctest(test_dir, test_obj, buildenv):
     """Runs a single test, comparing output and RC to expected output and RC.
 
     Raises an error if input can't be read, executable fails, or output/RC
     are not as expected. Error is caught by bctester() and reported.
     """
     # Get the exec names and arguments
-    execprog = buildenv["BUILDDIR"] + "/src/" + testObj['exec'] + buildenv["EXEEXT"]
-    execargs = testObj['args']
+    execprog = os.path.join(buildenv["BUILDDIR"], "src", test_obj["exec"], buildenv["EXEEXT"])
```

Comment:
The last concatenation is string concatenation.

You can check if the tests pass locally by `make check`

Reply:
Thanks I just fixed it, Sorry for my careless mistake